pkg/liqoctl/install/aks/aks_test.go

```diff
@@ -53,6 +53,7 @@ var _ = Describe("Extract elements from AKS", func() {
 		cmd := &cobra.Command{}
 
 		GenerateFlags(cmd)
+		cmd.Flags().String("cluster-labels", "", "Cluster Labels to append to Liqo Cluster, supports '='.(e.g. --cluster-labels key1=value1,key2=value2)")
 		cmd.Flags().String("cluster-name", "", "Name to assign to the Liqo Cluster")
 		cmd.Flags().String("reserved-subnets", "", "")
```

Comment:
Do you mean the description of the flag? If so, it's because I saw that the "cluster-name" flag's decription was given, and I just wanted to stick to the same format.

Reply:
I mean the whole flags definition which is copied in the tests

Reply:
IIRC, I added them since tests would break up, however I ran again the tests and I don't see any test failing, not to mention I don't see such flags used anywhere in the tests in the first place—I just rebased this PR to master, maybe the old code worked differently.